apps/store/src/components/Header/HeaderMenu/HeaderMenu.css.ts

```diff
@@ -29,15 +29,23 @@ export const backLink = style([
   },
 ])
 
-export const backWrapper = style({
-  backgroundColor: tokens.colors.buttonSecondary,
-  borderRadius: tokens.radius.xxs,
-  padding: tokens.space.xxs,
+export const backWrapper = style([
+  sprinkles({ display: 'flex', alignItems: 'center', justifyContent: 'center' }),
+  {
+    width: '2rem',
+    height: '2rem',
+    backgroundColor: tokens.colors.buttonSecondary,
+    borderRadius: tokens.radius.xxs,
+
+    ':hover': {
+      backgroundColor: tokens.colors.buttonSecondaryHover,
+    },
```

Comment:
If it should always be a square we can leverage `aspect-ratio`:

```suggestion
export const backWrapper = style([
  sprinkles({ display: 'flex', alignItems: 'center', justifyContent: 'center' }),
  {
    width: '2rem',
    aspectRatio: '1 / 1',
    backgroundColor: tokens.colors.buttonSecondary,
    borderRadius: tokens.radius.xxs,

    ':hover': {
      backgroundColor: tokens.colors.buttonSecondaryHover,
    },
    
    ...responsiveStyles({
      lg: { width: '2.5rem' },
    }),
```

Reply:
Yeah, I like that! 👌